doc/api/stream.md

```diff
@@ -1652,8 +1652,8 @@ const { StringDecoder } = require('string_decoder');
 class StringWritable extends Writable {
   constructor(options) {
     super(options);
-    const state = this._writableState;
-    this._decoder = new StringDecoder(state.defaultEncoding);
+    const encoding = options ? options.defaultEncoding : 'utf8';
+    this._decoder = new StringDecoder(encoding);
```

Comment:
Nit: it could be just `options && options.defaultEncoding`.

Reply:
Ah, yeah. Good point.